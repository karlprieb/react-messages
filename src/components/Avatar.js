import React from 'react'
import PropTypes from 'prop-types'

import './avatar.css'

const Avatar = ({ src }) => (
  <img src={src} alt='avatar' className='message-avatar' />
)

Avatar.propTypes = {
  src: PropTypes.string.isRequired
}

export default Avatar
