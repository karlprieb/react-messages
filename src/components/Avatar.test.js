import React from 'react'
import Avatar from './Avatar'

import avatarImage from '../assets/avatar.jpg'

describe('Avatar component', () => {
  it('should render avatar image', () => {
    const wrapper = shallow(<Avatar src={avatarImage} />)
    expect(wrapper).toMatchSnapshot()
  })
})
