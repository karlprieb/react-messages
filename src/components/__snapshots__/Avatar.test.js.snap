// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar component should render avatar image 1`] = `
<img
  alt="avatar"
  className="message-avatar"
  src="avatar.jpg"
/>
`;
